bitrise.yml

```diff
@@ -3,36 +3,33 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 app:
   envs:
-  - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-simple-objc-with-uitest.git
-  - SAMPLE_APP_BRANCH: renamed-scheme
+  - ORIG_BITRISE_SOURCE_DIR: $BITRISE_SOURCE_DIR
 
 workflows:
   test:
     before_run:
     - audit-this-step
     - go-tests
-    steps:
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            rm -rf "$BITRISE_SOURCE_DIR/_tmp"
-            mkdir -p "$BITRISE_SOURCE_DIR/_tmp"
-    - change-workdir:
-        title: Switch working dir to _tmp dir
-        run_if: true
-        inputs:
-        - path: "$BITRISE_SOURCE_DIR/_tmp"
-        - is_create_path: true
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ev
-            git clone --single-branch -b "$SAMPLE_APP_BRANCH" "$SAMPLE_APP_URL" .
-    - certificate-and-profile-installer:
     after_run:
+    - test_objc_all
+
+  test_swiftpm:
```

Comment:
`test_swiftpm` WF is not called. 🤔

Reply:
Had an issue running the workflow due to codesigning issues.